modules/tinymce/src/core/main/ts/api/util/XHR.ts

```diff
@@ -59,10 +59,152 @@ const XHR: XHR = {
 
   /**
    * Sends a XMLHTTPRequest.
-   * Consult the TinyMCE Documentation for details on what settings this method takes.
    *
    * @method send
-   * @param {Object} settings Object will target URL, callbacks and other info needed to make the request.
+   * @param {Object} settings An object containing the target URL, callbacks, and other information needed to make the request. For information on valid settings, see the table below.<br /><br />
+   * <div>
+   * <table>
+   * <thead>
+   * <tr style="text-align: center;">
+   * <th>Setting</th>
+   * <th>Required/<br />Optional</th>
+   * <th>Type</th>
+   * <th>Description</th>
+   * </tr>
+   * </thead>
+   * <tbody>
+   * <tr>
+   * <td>
+   * <p><code>url</code></p>
+   * </td>
+   * <td>Required</td>
+   * <td>
+   * <p><code>string</code></p>
+   * </td>
+   * <td>Address to send the request to, such as the back-end server.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>async<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>boolean</code></p>
+   * </td>
+   * <td>When <code>false</code>, the request will be synchronous. Set to <code>true</code> by default.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>content_type<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>string</code></p>
+   * </td>
+   * <td>Used to define the mime-type of the data. Similar to
+   * <a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/overrideMimeType">
+   * the XMLHttpRequest.overrideMimeType method</a>.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>crossDomain<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>boolean</code></p>
+   * </td>
+   * <td>When <code>true</code>, the withCredentials property will be set to <code>true</code>.
+   * For information on the withCredentials property, see:
+   * <a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials">
+   * MDN Web Docs - XMLHttpRequest.withCredentials</a>.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>data<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>any</code></p>
```

Comment:
need to update with `Document | BodyInit`

Reply:
customers may not know what `BodyInit` means fwiw. It might be better to do something like `Document | Blob | FormData | string`

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5924/commits/5e2449ba7b7799f66d922db66adb57acf7ec857e